src/JuliaFormatter.jl

```diff
@@ -268,8 +272,9 @@ function format_file(
         error("$filename must be a Julia (.jl) source file")
     end
     verbose && println("Formatting $filename with indent = $indent, margin = $margin")
-    str = read(filename) |> String
+    str = String(read(filename))
     str = format_text(str, indent = indent, margin = margin, always_for_in = always_for_in)
+    !endswith(str, "\n\n") && (str *= "\n")
```

Comment:
```suggestion
    !endswith(str, "\n") && (str *= "\n")
```

Would that work then?

Reply:
Yes, that seems right 👍